tests/test_isrTask.py

```diff
@@ -183,8 +183,8 @@ def test_flatCorrection(self):
         self.task.flatCorrection(self.inputExp, flatIm)
         statAfter = computeImageMedianAndStd(self.inputExp.image[self.amp.getBBox()])
         self.assertGreater(statAfter[1], statBefore[1])
-        self.assertFloatsAlmostEqual(statAfter[1], 147417.36, atol=1e-2)
-        self.assertFloatsAlmostEqual(statBefore[1], 148.28436, atol=1e-2)
+        self.assertFloatsAlmostEqual(statAfter[1], 147407.02, atol=1e-2)
```

Comment:
And here (and the ones below, etc)

Reply:
The "science" images used for these tests are generated with `doAddFringe=True`.  To better test the multi-fringe code, the wavelength of the function was cut by a factor of 3, which results in the fringe component being significant over a smaller area.  This gives slightly smaller median and sigma values (as more area is not influenced by the fringe pattern).